### DISCUSSION TITLE
Problems window custom icons

### DISCUSSION THREAD

**Discussion by salil-sharma-ar** [microsoft/vscode-discussions#1085] [Extension Development QnA] [answered]
Is it possible to change/customize icons based on Severity in the problems window of VSCode?

<img width="55" alt="Screenshot 2024-03-06 at 3 08 42 PM" src="https://github.com/microsoft/vscode-discussions/assets/150223667/e57ee60b-266f-4cc6-9b52-5b329833b5db">

These icons do not really fit with the problems displayed by the extension I am currently working on. I cannot find anything in the API that points to this being a possibility.

**Comment by burkeholland** [comment, marked as answer]
I can't find anything that states that icons in the problem view are customizable.

**Reply by isidorn** [reply]
You can use product icon themes https://code.visualstudio.com/api/extension-guides/product-icon-theme

**Reply by isidorn** [reply]
i.e. your extension can contribute a product icon theme
